development/docker-compose-deps-nats.yml

```diff
@@ -49,7 +49,7 @@ services:
     volumes:
       - workflow_db:/var/lib/postgresql/data
     healthcheck:
-      test: ["CMD-SHELL", "pg_isready"]
+      test: ["CMD-SHELL", "pg_isready -q -d ${INFRAHUB_TASKMANAGER_DB_DATABASE:-prefect} -U ${INFRAHUB_TASKMANAGER_DB_USER:-postgres}"]
```

Comment:
To avoid the complaint from the YAML linter we should changes this to.
Same thing for the other files.

```yaml
     test: 
       - "CMD-SHELL" 
       - "pg_isready -q -d ${INFRAHUB_TASKMANAGER_DB_DATABASE:-prefect} -U ${INFRAHUB_TASKMANAGER_DB_USER:-postgres}"
```